proto/equal_test.go

```diff
@@ -37,6 +37,7 @@ import (
 	. "github.com/gogo/protobuf/proto"
 	proto3pb "github.com/gogo/protobuf/proto/proto3_proto"
 	pb "github.com/gogo/protobuf/proto/test_proto"
+	casttypepb "github.com/gogo/protobuf/test/casttype/combos/both"
```

Comment:
I would rather not import /test from the /proto package. I would rather have a stand-alone test in the /test package.

I didn't find any other casttype tests. So lets create a new package at:
./test/casttype/singles
We can then create specific casttype tests in that package which the 'combos' package doesn't cover.

So lets define a singles.proto with a message containing the ./test/castype.Bytes as a casttype.

Then we can just create our equal test; Instantiating two message with bytes casttypes and calling proto.Equal with them.

When defining a makefile for the ./test/casttype/singles/ package, make sure to use the right number of ```../``` for the protoc paths. And add the regeneration to the project root makefile.